src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -180,6 +180,16 @@ <h2 class="sectionTitle">
             <div class="fieldDescription checkboxFieldDescription">${EnableTrueHdHelp}</div>
         </div>
 
+        <div class="selectContainer">
+            <select is="emby-select" id="selectPreferredTranscodeVideoCodec" label="${LabelSelectPreferredTranscodeVideoCodec}">
+                <option value="">${Auto}</option>
+                <option value="h264">H264</option>
```

Comment:
```suggestion
                <option value="h264">AVC</option>
```
I think it is more consistent if AVC was used. Given that HEVC is also used for h265.

Reply:
Fair reason. I do not know why we actually have a difference between the naming of HEVC and AVC. I had a look and our media info even shows `AVC: Yes`. But to my understanding h264 is always AVC as it is the same but with a different name.

Reply:
I agree with using h264 as we use it on other places as well.